components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -577,6 +578,17 @@ def validate_webui_config(
     validate_port(f"{WEBUI_COMPONENT_NAME}.port", clp_config.webui.host, clp_config.webui.port)
 
 
+def validate_mcp_server_config(clp_config: CLPConfig, logs_dir: pathlib.Path):
+    if not logs_dir.exists():
+        raise ValueError(
+            f"{MCP_SERVER_COMPONENT_NAME} logs directory at {str(logs_dir)} is missing."
+        )
```

Comment:
let's follow these linter rules:
* https://docs.astral.sh/ruff/rules/raw-string-in-exception/
* https://docs.astral.sh/ruff/rules/f-string-in-exception/#f-string-in-exception-em102
```suggestion
        error_msg = f"{MCP_SERVER_COMPONENT_NAME} logs directory at {str(logs_dir)} is missing."
        raise ValueError(error_msg)
```